src/jshint.js

```diff
@@ -102,6 +103,11 @@ var JSHINT = (function () {
     extraModules = [],
     emitter = new events.EventEmitter();
 
+  var experimental = new Experimental(addlabel, advance, blockstmt, doFunction, error,
+    expression, function () { return funct; }, function() { return inblock; },
+    isEndOfExpr, nobreaknonadjacent, nolinebreak, optionalidentifier, peek, prefix,
+    warning, warningAt);
```

Comment:
![my eyes are bleeding](http://img2.wikia.nocookie.net/__cb20140226234947/vampirediaries/images/a/ac/Supernatural-811-LARP-and-the-Real-Girl-bleeding-eyes.jpg)

<3 :D